.github/workflows/azure-dev.yml

```diff
@@ -36,6 +36,7 @@ jobs:
                 files: '["**/*.ts"]'
               env:
                 CLIENT_SECRET_TOKEN: ${{ secrets.AZURE_ADAPTIVECMS_CLIENT_SECRET }}
+                DB_CONNECTION_TOKEN: ${{ secrets.AZURE_ADPATIVECMS_DB_CONNECTION_STRING }}
```

Comment:
Is this typo ok? 'Adpative'

Reply:
Oops I misspelled the actual Github secret. Fixed.